tests/Fixer/Alias/ArrayPushFixerTest.php

```diff
@@ -273,7 +273,7 @@ public static function provideFixCases(): iterable
     /**
      * @dataProvider provideFixPre80Cases
      *
-     * @requires PHP <8.0
+     * @requires PHP < 8.0
```

Comment:
```suggestion
     * @requires PHP <8.0
```

CS seems more natural to me, it is what (`PHP`) and constraint (`<8.0`), like in composer, thus there should be no space between the operator.

Reply:
> to me

Have you ever considered yourself not to be the center of the universe? PHPUnit docs [have](https://docs.phpunit.de/en/11.5/annotations.html#requires) a space and those are PHPUnit annotations.

Reply:
Is that good or bad?

Reply:
What is good or bad? I meant I want to follow the format of what is in PHPUnit documentation rather than your personal preference.